wagtail/tests/test_management_commands.py

```diff
@@ -728,6 +728,19 @@ def test_purge_revisions_with_date_cutoff(self):
         # revision is now older than 30 days, so should be deleted
         self.assertRevisionNotExists(old_revision)
 
+    def test_purge_revisions_protected_error(self):
+        revision_old = self.object.save_revision()
+        PurgeRevisionsProtectedTestModel.objects.create(revision=revision_old)
+        revision_purged = self.object.save_revision()
+        revision_latest = self.object.save_revision()
+        PurgeRevisionsProtectedTestModel.objects.create(revision=revision_latest)
```

Comment:
There's no need to attach the latest revision to `PurgeRevisionsProtectedTestModel` as otherwise it's ambiguous whether the test ensures it's not removed due to it being the latest revision or due to `PROTECT`.

```suggestion
```